src/content/docs/guides/run-a-holesky-node.md

```diff
@@ -40,23 +40,13 @@ COMPOSE_FILE=lighthouse-cl-only.yml:geth.yml:el-shared.yml
 
 Keep in mind this is **not encrypted**, so you should not expose it to the internet. eth-docker offers a few other options that you can read about [here](https://eth-docker.net/Usage/Advanced#sharing-rpc-and-rest-ports).
 
-### 4. Enable archive node
-
-Also in the `.env` file, set the value `ARCHIVE_NODE` to `true`:
-
-```txt "true"
-ARCHIVE_NODE=true
-```
-
-This is no longer required to run a Taiko node.
```

Comment:
@davidtaikocha @cyberhorsey is the default behavior of the prover to only work on blocks within the last 128 blocks? i just want to make sure that removing this suggestion won't error out the default prover behavior.

otherwise i can leave a comment that it is only required (occasionally) for running a prover.